contracts/access/rbac/MinterRole.sol

```diff
@@ -8,29 +8,25 @@ contract MinterRole {
 
   Roles.Role private minters;
 
-  constructor(address[] _minters) public {
-    minters.addMany(_minters);
+  constructor() public {
+    minters.add(msg.sender);
   }
 
-  function transferMinter(address _account) public {
```

Comment:
I'm a bit reluctant to remove `transfer` because it means in order to transfer now one has to do two transactions. On the other hand, this may be the 2-step transfer we had been wanting. (!!) :slightly_smiling_face:

Reply:
We can always add it later :tada: